src/ontology/foodon-edit.owl

```diff
@@ -56109,16 +56110,6 @@ AnnotationAssertion(oboInOwl:hasDbXref obo:FOODON_03420120 "http://www.langual.o
 AnnotationAssertion(rdfs:label obo:FOODON_03420120 "whole animal, skinned, defeathered or scaled, eviscerated"@en)
 SubClassOf(obo:FOODON_03420120 obo:FOODON_03420173)
 
-# Class: obo:FOODON_03420122 (nonmeat part of animal)
```

Comment:
It is sort of undocumented, but all the anatomical parts are actually part of foodon-edit.owl , so I think nonmeat part of animal falls in that category.  Its just the products that go into robot table or foodon_product_import.owl .  So could we move this back?